### PR TITLE
fix(utils): Parameter 'e' implicitly has an 'any' type

### LINKED ISSUE #105: TS7006: Parameter 'e' implicitly has an 'any' type.
In version 0.8.3 of this package, a change was made to line 39 of the utils.js file.
```javascript
() => { throw({ loadChunkError: true }); }
```
was changed to:
```javascript
e => { throw({ loadChunkError: true, details: e }); }
```
I appreciate the intent of the modification, however, it is causing our tslint checks to fail with the error:
```
TS7006: Parameter 'e' implicitly has an 'any' type.
```
I'm not sure how we can resolve the problem we are seeing while maintaining the error reporting capabilities that you added. I'm open to some discussion about it and would even be willing to do any of the work that is needed to make a fix. Including validating it on our code.

Thank you for taking the time to review this issue report and for helping us in getting it resolved,
Clayton Hunt

### PR DESCRIPTION
**Pull request by ClaytonHunt**
This fix is to add an explicit type to error function argument.

Closes #105

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<a name="0.8.4"></a>
```
Comment (brandonroberts) [inline CHANGELOG.md]: Remove these changes.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (brandonroberts) [inline package.json]: Remove these changes. I'll will bump the version when I tag

File: spec/index.spec.js [hunk 3]
```diff
@@ -172,7 +172,7 @@ describe('Loader', function() {
   it ('should return a loadChildren System.import statement', function() {
     var result = [
       'loadChildren: function() { return System.import(\'./path/to/file.module\')',
-      '  .then(module => module[\'FileModule\'], e => { throw({ loadChunkError: true, details: e }); }) }'
+      '  .then(module => module[\'FileModule\'], e: any => { throw({ loadChunkError: true, details: e }); }) }'
```
Comment (brandonroberts) [inline spec/index.spec.js]: Wrap the `e: any` in parenthesis

**Comment by brandonroberts** [comment]
Thanks @ClaytonHunt for the PR! I left some feedback. Will you make the changes so this can land? Then I'll publish another patch release

**Comment by ClaytonHunt** [comment]
Should be updated, removed package and changelog modifications

**Pull request merged**